wordpress-mobile/WordPress-Android
# Implements: the UI tweaks in Pages and Posts list

## Part of
#19540 

## Description
This PR implements the design changes for Pages list and Posts list

## Explanation of Changes
- Changes the Add pages icon to use plus icon used in posts list
- Increases the padding between the more icon in pages and posts list to increase the touch target size 
- Decreases the min height of the pages list and adds a vertical padding in the pages list row, date header in scheduled and Home page row


## To test:
1. Login to the jetpack app with a site having some pages and posts in each status(Published, Draft, Scheduled, Trashed)
2. Go to Pages
3. Verify that the design is as expected
4. Verify the Add page icon is as expected
5. Go to Posts 
6. Verify that the design is as expected

## Regression Notes
1. Potential unintended areas of impact
Posts list and Pages list are not correct

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)


PR submission checklist:

UI Changes testing checklist:
